packages/block-library/src/image/edit.js

```diff
@@ -80,6 +80,7 @@ export function ImageEdit( {
 	insertBlocksAfter,
 	noticeOperations,
 	onReplace,
+	context,
```

Comment:
Instead of setting the `allowAlign` and `allowResize` variables below, you can do this instead:
```suggestion
	context: { allowAlign = true, allowResize = true },
```

Reply:
done.